src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+import { useSearchParams } from 'react-router-dom';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [searchParams] = useSearchParams();
+  const query = searchParams.get('query') || null;
+  const centuries = searchParams.getAll('centuries') || null;
```

Comment:
Ensure that existing search parameters are preserved when updating `centuries`. This will prevent other parameters like `sex` or `query` from being unintentionally removed.